modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/utilities/dataRenderers.js

```diff
@@ -12,9 +12,19 @@
  * details.
  */
 
-import dataRenderers from '../data_renderers/index';
+import dataRenderers, {inputRenderers} from '../data_renderers/index';
```

Comment:
Just a question, I see we have a `utilities` directory (or package), is there any reason why it's called like that and not `utils`?

Reply:
Actually not so I'll proceed with renaming it! :)